test/test-misc.jl

```diff
@@ -0,0 +1,3 @@
+println("------------------------------------------------
+----------------- Misc. Tests ------------------")
+@test getiftversion() == "0.2.0"
```

Comment:
```suggestion
@test getiftversion() == "0.2.0"
@test getiftversion() != "unknown"
```

Maybe adding this will satisfy codecov? Codecov is so demanding :)

Reply:
I think it's the try catch block inside the `__init__` function that is not "getting tested". And I don't think there is a way to run that block unless I undo the change in `test/Project.toml`!